src/core/gosling.schema.ts

```diff
@@ -513,16 +833,26 @@ export interface DisplaceTransform {
     type: 'displace';
     // We could support different types of bounding boxes (e.g., using a center position and a size)
     boundingBox: {
-        startField: string; // The name of a quantitative field that represents the start position
-        endField: string; // The name of a quantitative field that represents the end position
-        padding?: number; // The padding around visual lements. Either px or bp
-        isPaddingBP?: boolean; // whether to consider `padding` as the bp length.
-        groupField?: string; // The name of a nominal field to group rows by in prior to piling-up
+        /** The name of a quantitative field that represents the start position */
+        startField: string;
+
+        /** The name of a quantitative field that represents the end position */
+        endField: string;
+
+        /** The padding around visual lements. Either px or bp */
+        padding?: number;
+
+        /** whether to consider `padding` as the bp length */
+        isPaddingBP?: boolean;
+
+        /** The name of a nominal field to group rows by in prior to piling-up */
+        groupField?: string;
     };
+    /** A string that specifies the type of diseplancement, one of `"pile"` and `"spread"`  */
     method: DisplacementType;
     newField: string;
 
-    // "pile" specific parameters (TODO: make this a separate interface)
+    /** Specify maximum rows to be generated (default has no limit) */
     maxRows?: number; // Specify maximum rows to be generated (default: `undefined` meaning no limit)
```

Comment:
Should we remove the original comment which is now duplicated?

```suggestion
    maxRows?: number;
```